natlas-agent/natlas/utils.py

```diff
@@ -1,11 +1,13 @@
 import ipaddress
 import os
 import shutil
+from pathlib import Path
 
 from natlas import logging
-
+from config import Config
 
 utillogger = logging.get_logger("Utilities")
+conf = Config()
```

Comment:
I agree, there are a few places where it's harder to pass around a single copy because modules aren't classes so I can't just initialize one copy of them that shares a common state. I could update every function to take a copy of the config, but that doesn't really make sense in a number of places.

In the Agent rewrite for plugin architecture, I'll write it such that basically all of this stuff is encapsulated in classes and we won't use module-scoped copies of the config - instead initializing a class with the config and keeping the config and logger as properties of the class instead of properties of the module.